website/docs/reference/global-configs/warnings.md

```diff
@@ -2,129 +2,206 @@
 title: "Warnings"
 id: "warnings"
 sidebar: "Warnings"
+toc_max_heading_level: 2
+intro_text: "dbt's warnings can be converted to errors using --warn-error flag or WARN_ERROR config, with granular control through options."
 ---
 
-Enabling `WARN_ERROR`config or setting the `--warn-error` flag will convert dbt warnings into errors. Any time dbt would normally warn, it will instead raise an error. Examples include `--select` criteria that selects no resources, deprecations, configurations with no associated models, invalid test configurations, or tests and freshness checks that are configured to return warnings.
+Enabling `WARN_ERROR` config or setting the `--warn-error` flag will convert dbt warnings into errors. Any time dbt would normally warn, it will instead raise an error. Examples include `--select` criteria that selects no resources, deprecations, configurations with no associated models, invalid test configurations, or tests and freshness checks that are configured to return warnings.
 
  You would commonly use the `--warn-error` flag to promote test warnings from `severity: warn` to errors, but it actually affects all warning types, including:
 * Test warnings (for example, `LogTestResults`)
```

Comment:
remove "for example" this is the **exact** event name you should use for `--warn-error-options`

maybe helpful to show example as part of the flag - `--warn-error-options '{"error": ["LogTestResults"]}'`